vite-plugin-laravel/src/types.ts

```diff
@@ -0,0 +1,44 @@
+import type { SSROptions } from 'vite'
```

Comment:
Maybe this for TS 4.5+ ?
```typescript
import { type SSROptions } from 'vite'
```

Reply:
Didn't know about that, cool! I won't change it now because I'm not sure how `esbuild` plays with it, but that's nice to know and I'll surely experiment later, thanks 👍